pkg/yurthub/cachemanager/cache_manager_test.go

```diff
@@ -2080,54 +2080,55 @@ func TestQueryCacheForList(t *testing.T) {
 				},
 			},
 		},
-		"list runtimeclass": {
-			keyPrefix:  "kubelet/runtimeclasses",
-			noObjs:     true,
-			userAgent:  "kubelet",
-			accept:     "application/json",
-			verb:       "GET",
-			path:       "/apis/node.k8s.io/v1beta1/runtimeclasses",
-			namespaced: false,
-			expectResult: struct {
-				err  bool
-				rv   string
-				data map[string]struct{}
-			}{
-				data: map[string]struct{}{},
-			},
-		},
-		"list pods and no pods in cache": {
-			keyPrefix:  "kubelet/pods",
-			noObjs:     true,
-			userAgent:  "kubelet",
-			accept:     "application/json",
-			verb:       "GET",
-			path:       "/api/v1/pods",
-			namespaced: false,
-			expectResult: struct {
-				err  bool
-				rv   string
-				data map[string]struct{}
-			}{
-				err: true,
-			},
-			queryErr: storage.ErrStorageNotFound,
-		},
-		"list resources not exist": {
-			userAgent:  "kubelet",
-			accept:     "application/json",
-			verb:       "GET",
-			path:       "/api/v1/nodes",
-			namespaced: false,
-			expectResult: struct {
-				err  bool
-				rv   string
-				data map[string]struct{}
-			}{
-				err: true,
-			},
-			queryErr: storage.ErrStorageNotFound,
-		},
```

Comment:
we need to keep these test cases.

Reply:
The PR submitted recently recovered this test case.